addons/mail/static/src/discuss/core/common/thread_actions.js

```diff
@@ -10,6 +11,18 @@ import { _t } from "@web/core/l10n/translation";
 import { usePopover } from "@web/core/popover/popover_hook";
 
 threadActionsRegistry
+    .add("attachments", {
+        condition: (component) =>
+            component.thread?.model === "discuss.channel" &&
+            (!component.props.chatWindow || component.props.chatWindow.isOpen),
+        component: AttachmentPanel,
+        icon: "fa fa-fw fa-paperclip",
+        iconLarge: "fa fa-fw fa-lg fa-paperclip",
+        name: _t("Show Attachment List"),
+        nameActive: _t("Hide Attachment List"),
```

Comment:
Nit-picking: _"Show/Hide Attachments"_ is more brief and to the point. "List" adds another word, makes it less readable overall and has no improvement over plural "Attachments"